tests/integrationv2/providers.py

```diff
@@ -607,6 +619,26 @@ def setup_server(self):
         return cmd_line
 
 
+class SSLv3Provider(OpenSSL):
+    def __init__(self, options: ProviderOptions):
+        OpenSSL.__init__(self, options)
+        self._override_libssl(options)
+
+    def _override_libssl(self, options: ProviderOptions):
+        install_dir = os.environ["OPENSSL_1_0_2_INSTALL_DIR"]
+
+        override_env_vars = dict()
+        override_env_vars["PATH"] = install_dir + "/bin"
+        override_env_vars["LD_LIBRARY_PATH"] = install_dir + "/lib"
```

Comment:
Normally over-writing this is fraught with peril, but it looks like the framework gets away with it, so long as the [process/ManagedProcess](https://github.com/aws/s2n-tls/blob/8ee7a3cb07dbd5020310023ad12e2acb151f1b09/tests/integrationv2/processes.py#L329C65-L329C73) code only runs one binary.